airbyte-integrations/connectors/source-mailchimp/source_mailchimp/streams.py

```diff
@@ -33,13 +33,15 @@
 
 
 class MailChimpStream(HttpStream, ABC):
-    url_base = "https://us2.api.mailchimp.com/3.0/"
+    url_base = ""
```

Comment:
We can use a [dynamic property](https://docs.airbyte.io/contributing-to-airbyte/python/concepts/python-concepts#properties)

```suggestion
    @property
    def url_base(self) -> str:
        return f"https://{self.data_center}.api.mailchimp.com/3.0/"
```